samples/modules/mctp/mctp_endpoint/README.rst

```diff
@@ -0,0 +1,43 @@
+.. zephyr:code-sample:: mctp_endpoint_sample
+   :name: MCTP Endpoint Sample
+
+   This sample shows how to create an MCTP endpoint (a reactive node in a
+   network of MCTP nodes) over UART. MCTP is a multiplexed messaging protocol
+   that works over a variety of transports (i2c, i3c, uart, pcie, etc).
```

Comment:
This is meant to be the (very) short description that shows up in the code sample catalog (https://docs.zephyrproject.org/latest/samples/index.html). Suggested rewording as per code sample README template below (for example, it shouldn't be starting with "This sample shows...", but you may want to move some of the original text to the Overview section as it probably contains useful bits

```suggestion
   Create an MCTP endpoint over UART.
```